code/go/0chain.net/blobbercore/stats/blobberstats.go

```diff
@@ -306,21 +306,14 @@ func (bs *BlobberStats) loadAllocationStats(ctx context.Context) {
 		bs.AllocationStats = append(bs.AllocationStats, as)
 	}
 
-	if err = rows.Err(); err != nil && err != sql.ErrNoRows {
+	if err = rows.Err(); err != nil && !errors.Is(err, sql.ErrNoRows) {
 		Logger.Error("Error in scanning record for blobber stats",
 			zap.Error(err))
 		return
 	}
 
-	var count int64
```

Comment:
not sure I understand this change, can you elaborate please

Reply:
I changes this to fix the empty pages issue in diagnostic pages.